ecl/network/v2/fic_gateways/testing/request_test.go

```diff
@@ -0,0 +1,69 @@
+package testing
+
+import (
+	"fmt"
+
+	"github.com/nttcom/eclcloud/ecl/network/v2/fic_gateways"
+	"github.com/nttcom/eclcloud/pagination"
+
+	"net/http"
+	"testing"
+
+	fake "github.com/nttcom/eclcloud/ecl/network/v2/common"
+	th "github.com/nttcom/eclcloud/testhelper"
+)
+
+func TestList(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+
+	th.Mux.HandleFunc("/v2.0/fic_gateways", func(w http.ResponseWriter, r *http.Request) {
+		th.TestMethod(t, r, "GET")
+		th.TestHeader(t, r, "X-Auth-Token", fake.TokenID)
+
+		w.Header().Add("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+
+		fmt.Fprintf(w, ListResponse)
+	})
+
+	client := fake.ServiceClient()
+	count := 0
+
+	fic_gateways.List(client, fic_gateways.ListOpts{}).EachPage(func(page pagination.Page) (bool, error) {
+		count++
+		actual, err := fic_gateways.ExtractFICGateways(page)
+		if err != nil {
+			t.Errorf("Failed to extract FIC Gateways: %v", err)
+			return false, nil
+		}
+		th.CheckDeepEquals(t, ExpectedFICGatewaySlice, actual)
+
+		return true, nil
+	})
+
+	if count != 1 {
+		t.Errorf("Expected 1 page, got %d", count)
+	}
+}
+
+func TestGet(t *testing.T) {
```

Comment:
I think this method name should be added suffix like "FICGateway". The method name "TestGet" is output to the test log and we cannot recognize the test target.